src/Node/ImportNode.php

```diff
@@ -25,9 +25,6 @@
 #[YieldReady]
 class ImportNode extends Node
 {
-    /**
-     * @param bool $global
-     */
     public function __construct(AbstractExpression $expr, AbstractExpression|AssignTemplateVariable $var, int $lineno)
     {
         if (!\is_bool(\func_num_args() > 3)) {
```

Comment:
this check is broken. `\func_num_args() > 3` is always a boolean

Reply:
Indeed. I changed to 
```
if (\func_num_args() > 3) {
```

This would have been reported by phpstan on a higher level I think.